### PR TITLE
fix #572: default to en when no signup_language

### LINKED ISSUE #572: acceptLanguage error on HIBP callback requests
STR:
`curl --verbose --data "{\"breachName\": \"BabyNames\", \"hashPrefix\":\"12eeb9\", \"hashSuffixes\":[\"98fe0c7600b3a3aacf518aa760b1dbd775\"]}" -H "Content-Type: application/json" -H "Authorization: Bearer <token for heroku>" https://fx-breach-alerts.herokuapp.com/hibp/notify`

Expected Results:
```
< HTTP/1.1 200 OK
...
{"info":"Notified subscribers of breach."}
```

Actual Results:
```
< HTTP/1.1 500 Internal Server Error
...
{"message":"Argument must be a string"}
```

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Comment by groovecoder** [comment]
Also updating `merge` module to `1.2.1` to fix sec vuln.

File: controllers/hibp.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ async function notify (req, res) {
 
     const email = subscriber.email;
     // need to create unsubscribe button for these
-    const requestedLanguage = acceptedLanguages(subscriber.signup_language);
+    const requestedLanguage = subscriber.signup_language ? acceptedLanguages(subscriber.signup_language) : "";
```
Comment (pdehaan) [inline controllers/hibp.js]: Why default to empty string instead of of "en" here?

File: package-lock.json [hunk 2]
```diff
@@ -7006,9 +7006,9 @@
       }
     },
     "merge": {
-      "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/merge/-/merge-1.2.0.tgz",
-      "integrity": "sha1-dTHjnUlJwoGma4xabgJl6LBYlNo="
+      "version": "1.2.1",
+      "resolved": "https://registry.npmjs.org/merge/-/merge-1.2.1.tgz",
+      "integrity": "sha512-VjFo4P5Whtj4vsLzsYBu5ayHhoHJ0UqNm7ibvShmbmoz7tGi0vXaoJbGdB+GmDMLUdg8DpQXEIeVDAe8MaABvQ=="
```
Comment (pdehaan) [inline package-lock.json]: Mildly confused by this.
I don't see how this was fixed, exactly.
Shouldn't we be updating dependency versions in package.json to resolve vuln **merge** versions? Wouldn't this just revert if somebody clobbered the package-lock.json file and regenerated it (to fix a merge conflict, etc)?

Shouldn't we just bump **jest** to the latest version?

File: controllers/hibp.js (hunk 1 above)
Comment (groovecoder) [reply controllers/hibp.js]: I thought I was cleverly avoiding hard-coding the default language, but it's hard-coded right below here anyway. Hmm ...

File: package-lock.json (hunk 2 above)
Comment (groovecoder) [reply package-lock.json]: Yeah I'm not sure how this fixed it either ... I just kept doing what `npm audit` told me to do until it passed.

**Comment by groovecoder** [comment]
Merging to fix master on Travis.

**Pull request merged**